packages/integration-tests/projects/transport/common.setup.js

```diff
@@ -19,9 +19,9 @@ const wait = ms =>
 
 const setup = async controller => {
     await controller.connect();
+    await controller.send({ type: 'bridge-stop' });
```

Comment:
just curious, why these changes? why adding stop and removing start?

Reply:
packages/integration-tests/projects/transport/tests/bridge.integration.js now runs against multiple bridge versions and it starts bridge in the test itself. See line 35. 
This setup is run in beforEach is it works like this:
- stop bridge
- do some setup
- start bridge version we want to test at the moment

Reply:
I see, thanks.